LanguageFeatures/Static-access-shorthand/non_ambiguity_A02_t03.dart

```diff
@@ -59,30 +57,62 @@ extension type ET(int value) {
 
 main() {
   var l1 = <C>[
-  ....one,
+  ....one, // `...` has type `Iterable<C>` but it has no static member `one`
```

Comment:
Perhaps:
```suggestion
  ....one, // `...` yields the context `Iterable` which has no static member `one`
```
Elements (including spread elements) don't have a type (they are not expressions), but they contribute to the determination of an element type for a list or set literal, and to a key or value type for a map literal, so it's somewhat confusing to talk about a spread element having a type. Also, `...` in itself is even further from having a type. The operand of a spread element does have a context type, and it does have a static namespace which is used for dot shorthand lookups.